index.html

```diff
@@ -0,0 +1,236 @@
+            This is a website for recipes made with Ruby on Rails
+          </p>
+          <ul class="project-stacks card-stacks">
+            <li>Ruby</li>
+            <li>Ruby on Rails</li>
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>ERB</li>
+            <li>RSPec</li>
+          </ul>
+          <a href="https://github.com/Spayco/katchy/" class="card-link link" target="_blank"
+            rel="noreferrer noopener">See project</a>
+        </div>
+      </div>
+
+    </div>
+    <div class="about-me">
+      <section class="about-me-section">
+        <h2>About me</h2>
+        <p class="about-me-text">
+          I am a skilled Full-Stack Web Developer who loves making pages with
+          beautiful UI. I have studied at Microverse, and I am experienced in
+          remote work.
+          <br />
+          I've also been a good helping hand to a lot of developers in
+          different communities!
+          <br />
+          if you'd like to get in contact about potential job opportunities,
+          please reach out!
+        </p>
+        <a href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+          class="link" target="_blank" rel="noreferrer noopener">
+          Get my resume
+        </a>
+        <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+      </section>
+      <hr />
+      <div class="about-me-section stacks">
+        <ul>
+          <li class="languages list-title">
+            Languages
+            <ul>
+              <li>JavaScript</li>
+              <li>TypeScript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>Sass</li>
+            </ul>
+          </li>
+        </ul>
+        <ul>
+          <li class="frameworks list-title">
+            Frameworks
+            <ul>
+              <li>React & Redux</li>
+              <li>Material-UI</li>
+              <li>Jest</li>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+            </ul>
+          </li>
+        </ul>
+        <ul>
+          <li class="skills list-title">
+            Skills
+            <ul>
+              <li>GitHub</li>
+              <li>CodePen</li>
+              <li>GitLab</li>
+              <li>Terminal</li>
+              <li>WSL</li>
+              <li>Remote Pair-Programming</li>
+              <li>Teamwork</li>
+              <li>Mentoring</li>
+              <li>Code Reviews</li>
+              <li>Mobile/Responsive Development</li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </main>
+
+  <section class="contact-form">
+    <div class="contact-form-header">
+      <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+    </div>
+    <form action="https://formspree.io/f/mbjqyrqj" method="POST">
```

Comment:
- [ ] I think the layout of the form is not built properly. As you can see in the pictures attached they are completely different. Kindly change the layout of the form to resemble the Figma design. 

<p align="center">&nbsp;
 
| Your output |   Figma design |
| ---------- | ----------- |
 | <img align="center" src="https://user-images.githubusercontent.com/12524453/175757963-e6dba106-b095-481d-9e76-cb0d8adf5efb.png" alt="The footer section of the figma design" width="410"/> | <img align="center" src="https://user-images.githubusercontent.com/12524453/175757929-8567cc62-af16-40a6-94e3-8dce5e3764bc.png" alt="The footer section of the figma design" width="410"/> |
</p>